thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -304,6 +307,12 @@ func checkFnConfigPathExistence(path string) error {
 }
 
 func (r *EvalFnRunner) preRunE(c *cobra.Command, args []string) error {
+	// Let users know that --include-meta-resources is not longer necessary
+	// since meta resources are included by default.
+	if r.IncludeMetaResources {
+		return fmt.Errorf("--include-meta-resources is no longer necessary because meta resources are now included by default")
```

Comment:
I like this user friendly message!

Reply:
Shall we return errors or moving forward? 

Returning errors could break some user auto-process, suggest giving a warning and moving forward instead.

Reply:
I thought about just giving warning first but given that the default behavior for `--include-meta-resources` ("false") is no longer supported so keeping the flag around didn't make sense, so had to take it out.